package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "keycloak-theme-govuk",
-  "version": "2.0.9",
+  "version": "2.0.20",
```

Comment:
Although the repo homepage says 2.0.19 is the latest, on the releases page, it says that the *.20 tag already exists. Perhaps this should be *.21?